.mergify.yml

```diff
@@ -97,3 +97,14 @@ pull_request_rules:
       label:
         add:
           - backport-skip
+  - name: notify the backport has not been merged yet
+    conditions:
+      - -merged
+      - -closed
+      - author=mergify[bot]
+      - schedule=Mon-Fri 06:00-10:00[Europe/Paris]
+      #- created-at<3 days ago
+    actions:
+      comment:
+        message: |
+          This pull request has not been merged yet. Could you please review it @{{assignee}}? 🙏
```

Comment:
![image](https://user-images.githubusercontent.com/2871786/141300419-ebebede1-2930-4926-bd69-4091a671b566.png)

It's a list of elements, therefore this should be changed